src/lambda_codebase/initial_commit/bootstrap_repository/adf-build/requirements.txt

```diff
@@ -2,8 +2,8 @@
 astroid~=2.9.0
 aws-sam-cli==1.36.0
 awscli==1.22.26
-boto3==1.20.26
-botocore==1.23.26
+boto3==1.21.31
```

Comment:
When I use the given versions within `requirements.txt` I get an error

```bash
ERROR: Cannot install -r adf-build/requirements.txt (line 4) and botocore==1.24.32 because these package versions have conflicting dependencies.
--
  
The conflict is caused by:
The user requested botocore==1.24.32
awscli 1.22.26 depends on botocore==1.23.26

To fix this you could try to:
  1. loosen the range of package versions you've specified
  2. remove package versions to allow pip attempt to solve the dependency conflict

ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts



ERROR: Cannot install -r adf-build/requirements.txt (line 4) and botocore==1.24.32 because these package versions have conflicting dependencies.
--
95 |  
96 | The conflict is caused by:
97 | The user requested botocore==1.24.32
98 | awscli 1.22.26 depends on botocore==1.23.26
99 |  
100 | To fix this you could try to:
101 | 1. loosen the range of package versions you've specified
102 | 2. remove package versions to allow pip attempt to solve the dependency conflict
103 |  
104 | ERROR: ResolutionImpossible: for help visit https://pip.pypa.io/en/latest/topics/dependency-resolution/#dealing-with-dependency-conflicts


```